projects/vdk-plugins/vdk-control-api-auth/src/vdk/plugin/control_api_auth/auth_request_values.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from enum import Enum
+
+
+class AuthRequestValues(Enum):
+    """
+    An enum used to the common values used in the redirect logic.
+    """
```

Comment:
Do you think it's a good idea to transfer also `ApiKeyAuthentication`, since it does make use of the Authentication class to instantiate it?
It contains the usage workflow; is it concurrent?

Reply:
Yes, `ApiKeyAuthentication` will also be transferred, alongside the other authentication flows, as separate PRs to make the changes easier to review.